### DISCUSSION TITLE
下载后添加emby提示 当前没有兼容的流。请稍后再试或联系您的管理员以获取详细信息。

### DISCUSSION THREAD

**Discussion by jacky940810** [jxxghp/MoviePilot#2619] [问题咨询]
![image](https://github.com/user-attachments/assets/7b97f934-d258-42c5-9eaa-232eec759c57)

**Comment by MoogleXX** [comment]
这是你浏览器不兼容emby这种在线流播放。和MP无关。你换其他浏览器或者安装html5播放器插件看看。
